src/main/java/com/google/devtools/build/lib/remote/http/HttpCacheClient.java

```diff
@@ -268,6 +274,13 @@ public void channelCreated(Channel ch) {
               SSLEngine engine = sslCtx.newEngine(ch.alloc(), hostname, port);
               engine.setUseClientMode(true);
               p.addFirst("ssl-handler", new SslHandler(engine));
+              // FIXME - This should use DEBUG level as it can potentially create a huge
+              //  amount of logs. However, this would require setting the log level
+              //  through the property java.util.logging.config.file.
+              //  Setting the value to INFO for simplicity. Will update it later if
+              //  we intend to keep this logging.
```

Comment:
Planning to change this. The amount of logs generated is unbearable. We'll need to enable it only when necessary.